esx_service/cli/vmdkops_admin.py

```diff
@@ -311,8 +311,10 @@ def list_of_values(string):
         given = string.split(',')
         for g in given:
             if g not in allowed:
-                msg = "invalid choice: {0} (choose from {1})".format(g,
-                                                                     allowed)
+                msg = (
+		   'invalid choices: {0} (choices must be a comma separated list of '
+                   'only the following words {1}. '  
```

Comment:
1. Please add newlines to beautify the message, It is way too long 
2. I agree with Prashant, would be good to drop the spaces (maybe run it via replace(", ", ","))